crates/turborepo-lib/src/engine/mod.rs

```diff
@@ -201,7 +197,19 @@ impl Engine<Built> {
                     }
                 }
 
-                Ok(is_persistent)
+                // check if the package for the task has that task in its package.json
+                let info = package_graph
+                    .workspace_info(&WorkspaceName::Other(task_id.package().to_string()))
```

Comment:
I'm assuming this is what's causing the crashes. `WorkspaceName::from` handles `//` correctly and maps it to `WorkspaceName::Root`
```suggestion
                    .workspace_info(&WorkspaceName::from(task_id.package().to_string()))
```

Reply:
TY